examples/services.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
I used this script to help me debug the issue in the `examples` integration suite. 
This script does 3 main things:

1. Compile the `xxx/main.go` binaries into binaries hosted in a temporary folder. The added value was that killing the servers became easier for detached processes (let me know if this is unclear I can explain better)
2. Redirect the logs of every service started to a temporary log folder.
3. Keep track of the pids of the services processes so it becomes easy to stop/restart them.

Reply:
Hey, would it not make more sense to run them with simple `go run main.go` command to keep lean and not need to create a bash script?
The idea is to open each of them in a separate terminal with `go run` and look at the logs in the terminal.

Reply:
I found a bit laborious to have 7 concurrent terminal windows opened and switch between and I only created the script because I found easier to tear down / bring up the instances as well as to refer to the right logs without having to 'search' through the 7 opened terminal windows. 
I understand that indeed there is the price of having another script therefore I can exclude this from the PR if you think it's not worth it.

Reply:
It makes sense my only concern was the maintenance.